style.css

```diff
@@ -0,0 +1,901 @@
+}
+
+.img-container {
+  width: 100%;
+  position: relative;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+  font-size: large;
+  font-weight: 500;
+}
+
+.center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.footer-h {
+  width: 50%;
+  margin: 1rem auto;
+  text-align: center;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: #3c3f46;
+}
+
+.footer-img {
+  display: block;
+  width: 20%;
+  margin: 5px auto;
+}
+
+.white {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.white p {
+  padding: 0.5rem;
+  width: 40%;
+  font-size: x-small;
+  font-weight: 600;
+}
+
+.microverse {
+  color: #ec5342;
+}
+
+.white a {
+  color: #ec5342;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+.logo {
+  display: none;
+}
+
+.top {
+  display: none;
+}
+
+/* ======================================================================= */
+
+/*  media queries desktop version */
+
+@media only screen and (min-width: 768px) {
+  .logo,
+  .menu-container {
+    display: none;
+  }
+
+  .top {
+    position: fixed;
+    top: 0;
+    z-index: 3;
+    background-color: #383a3f;
+    color: white;
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .top-logo {
+    margin-left: 2rem;
+    font-size: small;
+    padding-right: 1rem;
+  }
+
+  .top-logo:hover {
+    color: #ec5342;
+  }
+
+  .top-logo-img {
+    background-color: #383a3f;
+    color: #d3d3d3;
+    margin-left: 2rem;
+  }
+
+  .logo {
+    position: fixed;
+    top: 2px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+    z-index: 2;
+    opacity: 1;
+    background-color: whitesmoke;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    font-size: 12px;
+    height: 3rem;
+  }
+
+  .main-section {
+    margin-top: 4rem;
+    height: 100vh;
+    background-image: url(img/bg-headear.png);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .main-section-article {
+    margin-left: 3rem;
+    height: 100vh;
+    margin-top: 1rem;
+  }
+
+  .main-section-article h1 {
+    margin-top: 4rem;
+    font-size: 25px;
+    font-weight: 600;
+    text-transform: uppercase;
+    margin-left: 1rem;
+    margin-right: 1rem;
+  }
+
+  .main-section-article h2 {
+    font-size: 43px;
+    font-weight: 900;
+    width: 60%;
+    color: #ec5342;
+    margin-top: -1rem;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-bottom: 4rem;
+  }
+
+  .article-1 {
+    background-color: whitesmoke;
+    width: 54%;
+    height: 25vh;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-bottom: 1rem;
+    text-align: left;
+  }
+
+  .nav-list {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    text-align: center;
+    width: 60%;
+    margin: 0 2rem;
+  }
```

Comment:
- [ ] Kindly increase the font size of the nav items and decrease the gap. Please follow the design guidelines. 😄 

## Your Design:
![image](https://user-images.githubusercontent.com/92215878/192031012-8ca0f289-3f8b-4ab4-928e-07adf919ce87.png)

## Design Guidelines:
![image](https://user-images.githubusercontent.com/92215878/192031076-92497f1a-3dc5-445c-ba49-bdb006e7a0d0.png)